pages/common/yes.md

```diff
@@ -3,14 +3,14 @@
 > Output something repeatedly.
 > This command is commonly used to answer yes to every prompt by install commands (such as apt-get).
 
-- Repeatedly output "message":
+- Repeatedly output "message" :
```

Comment:
I think you have added a space that is not required here ^^

Reply:
```suggestion
- Repeatedly output "message":
```